### PR TITLE
Market buy tile fix -> Master

### PR DESCRIPTION
**Pull request by Mart500**
Added a SetOwner function in Tile and added extra to the BuyTile function in market.

### CODE REVIEW

**Comment by RobinStephenson** [comment]
@Mart500 which issues is this trying to fix

File: RoboticonColony/Assets/Code/Inventory.cs [hunk 1]
```diff
@@ -30,23 +29,17 @@ public Inventory(int money, int ore, int power, int roboticons)
         }
 
         Money = money;
-        Tiles = new List<Tile>();
-        Items = new Dictionary<ItemType, int>();
-        Items[ItemType.Ore] = ore;
-        Items[ItemType.Power] = power;
-        Items[ItemType.Roboticon] = roboticons;
+        tiles = new List<Tile>();
```
Comment (RobinStephenson) [inline RoboticonColony/Assets/Code/Inventory.cs]: Theses should be in PascalCase

File: RoboticonColony/Assets/Code/Inventory.cs [hunk 2]
```diff
@@ -30,23 +29,17 @@ public Inventory(int money, int ore, int power, int roboticons)
...
+        items[ItemType.Power] = power;
+        items[ItemType.Roboticon] = roboticons;
     }
 
     /// <summary>
     /// Creates an empty inventory instance, which holds all resources for a player or market.
     /// </summary>
     public Inventory() : this(0, 0, 0, 0) { }
-
-    //TEMP
```
Comment (RobinStephenson) [inline RoboticonColony/Assets/Code/Inventory.cs:45]: Theres a seperate issue to remove this when were ready to but at the moment michael needs it i think

**Comment by Mart500** [comment]
#57 @RobinStephenson

**Comment by Mart500** [comment]
also #68

**Review by RobinStephenson — CHANGES_REQUESTED** [review]

File: RoboticonColony/Assets/Code/Inventory.cs (hunk 2 above)
Comment (RobinStephenson) [reply RoboticonColony/Assets/Code/Inventory.cs:45]: Updating the issue description now to make this more clear. Sorry about that

**Comment by Mart500** [comment]
@RobinStephenson Fixed

**Review by RobinStephenson — APPROVED** [review]

**Comment by RobinStephenson** [comment]
@Mart500 you okay to do the merge conflicts?

**Pull request merged**